cookbook-react/src/scss/variables/_colors.scss

```diff
@@ -3,3 +3,7 @@ $black: #000;
 $red: #F00;
 $blue: #00F;
 $gray: #CCC;
+$alto: #DDD;
```

Comment:
Move these colors to the component's scss file. In a normal project this would be fine, but this is a library of different and unrelated components this colors aren't actually shared between them necessarily, so these colors are specific to your component in this context.